docs: add missing `}` and remove comma

morinokami opened the pull request:
### Description

- ~~The JSON code block's indentation has been corrected for better readability and proper JSON structure~~
- ~~Added the `inputs` property to `build` to match the actual [turbo.json](https://github.com/vercel/turborepo/blob/053a46031df7970059e458d3cc705f11b1daf313/examples/basic/turbo.json#L7) content~~
- Added a missing `}` to the JSON code block and removed a trailing comma

### Testing Instructions

<!--
  Give a quick description of steps to test your changes.
-->

ijjk commented:
Allow CI Workflow Run

- [ ] approve CI run for commit: 0d9c2e458f339a605d53b4cc3f0bfb08d785ec5a

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

ijjk commented:
Allow CI Workflow Run

- [ ] approve CI run for commit: 0d9c2e458f339a605d53b4cc3f0bfb08d785ec5a

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

anthonyshew commented:
Hi, thanks for the contribution!

It looks like we're missing the last `}` and the trailing comma does need to be removed, but we're happy with the indentation and configuration shown.

Can you update only for the `}` and comma? Thanks!

morinokami commented:
@anthonyshew Thanks for your feedback! I'll update the code accordingly.

morinokami commented:
@anthonyshew Hmm, I added the missing `}` and removed the comma without formatting, but it seems that `lint-staged` adds indentation on git commit. What should I do?

anthonyshew commented:
Hm, that sounds like a problem on our end. If you could `git commit -m "your message" --no-verify` to work around lint-staged for now, I'll look into that later.

morinokami commented:
@anthonyshew Thank you for the advice. I have committed using `--no-verify`.

anthonyshew reviewed (approved):
Great! Thank you!
